packngo.go

```diff
@@ -613,6 +622,12 @@ func checkResponse(r *http.Response) error {
 		return err
 	}
 
+	ct := responseContentType(r)
+	if ct != expectedAPIContentType {
+		errorResponse.SingleError = fmt.Sprintf("Unexpected Content-Type in response: %s", ct)
```

Comment:
Perhaps the most common reason we will receive a non-json response is when we get an HTML 5xx message.

I think it would be important to capture the 5xx message in these cases.

`"Unexpected Content-Type %q with Status %q"`:
   `Unexpected Content-Type "text/html" with Status "500 Server Unavailable"`

Reply:
will go